docs/security/trust/content_trust.md

```diff
@@ -240,17 +240,18 @@ these.
 ### Pull image content
 
 A common way to consume an image is to `pull` it. With content trust enabled, the Docker
-client only allows `docker pull` to retrieve signed images.
+client only allows `docker pull` to retrieve signed images. Let's try to pull the image
+you signed and pushed earlier:
 
 ```
-$  docker pull docker/seaside
+$  docker pull <username>/trusttest:testing
 Using default tag: latest
-Pull (1 of 1): docker/trusttest:latest@sha256:d149ab53f871
+Pull (1 of 1): <username>/trusttest:testing@sha256:d149ab53f871
 ...
-Tagging docker/trusttest@sha256:d149ab53f871 as docker/trusttest:latest
+Tagging <username>/trusttest@sha256:d149ab53f871 as docker/trusttest:testing
 ```
 
-The `seaside:latest` image is signed. In the following example, the command does not specify a tag, so the system uses
+In the following example, the command does not specify a tag, so the system uses
 the `latest` tag by default again and the `docker/cliffs:latest` tag is not signed.
```

Comment:
The `docker/cliffs` repository also doesn't exist.  Since the previous section signed in only `<username>/trusttest:testing`, maybe we can use `<username>/trusttest:latest` as the unsigned image?